circuits/cpp/src/aztec3/utils/types/native_types.hpp

```diff
@@ -39,34 +39,17 @@ struct NativeTypes {
     using VK = plonk::verification_key;
     using Proof = plonk::proof;
 
-    /// TODO: lots of these compress / commit functions aren't actually used: remove them.
-
-    // Define the 'native' version of the function `compress`, with the name `compress`:
-    static fr compress(const std::vector<fr>& inputs, const size_t hash_index = 0)
+    static crypto::GeneratorContext<curve::Grumpkin> get_generator_context(const size_t hash_index)
     {
-        return crypto::pedersen_commitment::compress_native(inputs, hash_index);
+        crypto::GeneratorContext<curve::Grumpkin> result;
```

Comment:
same comment as in the circuit_types file: https://github.com/AztecProtocol/aztec-packages/pull/2592/files#r1368467887